clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/security/AppengineJsonCredentials.groovy

```diff
@@ -20,8 +20,11 @@ import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
 import com.google.api.client.http.HttpTransport
 import com.google.api.client.json.JsonFactory
 import com.google.api.services.appengine.v1.AppengineScopes
+import com.google.auth.oauth2.GoogleCredentials
 import com.netflix.spinnaker.clouddriver.googlecommon.security.GoogleCommonCredentialUtils
+import groovy.transform.CompileStatic
 
+@CompileStatic
```

Comment:
Same comment about considering migrating to Java instead of using `@CompileStatic`.

Reply:
Have converted those 2 classes to java.